source/Title.js

```diff
@@ -0,0 +1,28 @@
+var Pixi = require("pixi.js")
+
+import Reference from "./Reference.js"
+
+export default class Junkership extends Pixi.Sprite {
```

Comment:
I think you forgot to change the class name here. It just so happens to still work because of the way we've been importing the classes.